analytics/pubstack/config_test.go

```diff
@@ -24,21 +25,20 @@ func TestFetchConfig(t *testing.T) {
 	server := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
 		defer req.Body.Close()
 		res.Write([]byte(configResponse))
-		res.WriteHeader(200)
 	}))
 
 	defer server.Close()
 
 	endpoint, _ := url.Parse(server.URL)
 	cfg, _ := fetchConfig(server.Client(), endpoint)
 
-	assert.Equal(t, cfg.ScopeID, "scopeId")
-	assert.Equal(t, cfg.Endpoint, "https://pubstack.io")
-	assert.Equal(t, cfg.Features[auction], true)
-	assert.Equal(t, cfg.Features[cookieSync], true)
-	assert.Equal(t, cfg.Features[amp], true)
-	assert.Equal(t, cfg.Features[setUID], false)
-	assert.Equal(t, cfg.Features[video], false)
+	assert.Equal(t, "scopeId", cfg.ScopeID)
```

Comment:
Testify assert statements should be in the format of "expected, actual" and many of these tests had them reversed leading to confusing failure messages.